### PR TITLE
override method 

### PR DESCRIPTION
**Pull request by SADSASD1**

### CODE REVIEW

File: User.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-public class User {
+public class  User extends Main {
```
Comment (tigran10) [inline User.java]: why extend main ?

File: User.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
     private String firstname = new String();
     private String lastname = new String();
     private String email = new String();
```
Comment (tigran10) [inline User.java]: no need to do `new String()`. 
`""` is default empty string

File: User.java [hunk 3]
```diff
@@ -49,11 +49,11 @@ public User(String firstname, String lastname, String email, String password) {
...
     }
-    public void saying(){
-        System.out.println(firstname);
-        System.out.println(lastname);
-        System.out.println(email);
-        System.out.println(password);
+
+    @Override
+    public String toString() {
+        return "\f firstname \n  lastname\n  email\n  password";
```
Comment (tigran10) [inline User.java]: the output of this command should be

```
firstname: FIRSTNAME_VALUE,
lastname: LASTNAME_VALUE
password: PASSWORD_VALUE
```

File: Main.java [hunk 4]
```diff
@@ -1,7 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        User us = new User("firstname","lastname","email","password");
-        us.saying();
-
+        User us = new User("firstname", "lastname", "email", "password");
```
Comment (tigran10) [inline Main.java]: use real values, like your name, your last name, your email, and some random password

File: User.java [hunk 5]
```diff
@@ -49,17 +49,13 @@ public User(String firstname, String lastname, String email, String password) {
...
-        System.out.println(email);
-        System.out.println(password);
-    }
-}
-
-
 
+    @Override
+    public String toString() {
+        return "\f FIRSTNAME_Ashot \n  LASTNAME_Aleqsanyan\n  EMAIL_aleqsanyan1996@mail.ru\n  PASSWORD_1234qwer";
```
Comment (tigran10) [inline User.java]: petqa firstname popoxakani arjeqy tpes, voch te static string. 

nuyny verabervuma mnacac popoxakannerin

File: Main.java [hunk 6]
```diff
@@ -1,7 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        User us = new User("firstname","lastname","email","password");
-        us.saying();
-
+        User us = new User("Ashot", "Aleqsanyan", "aleqsanyan1996@mail.ru", "1234qwer");
+        System.out.println(us.toString());
```
Comment (tigran10) [inline Main.java:4]: es methody vor kanchumes tes chishta tpum arjeqnery te che

Comment (tigran10) [reply Main.java:4]: ```User us = new User("Ashot", "Aleqsanyan", "aleqsanyan1996@mail.ru", "1234qwer");```


es methody ``` System.out.println(us.toString());``` kancheluc heto

petqa console-um tpvi 
```
name: Ashot
lastName: Alesanyan
email: aleqsanyan1996@mail.ru
pass: 1234qwer
```


Bayc vor poxem sen grem
```
User us = new User("Tigran", "Mnatsakanyan", "tigran@mna.com", "1234qwer");
System.out.println(us.toString());
```

petqa console-um tpvi 
```
name: Tigran
lastName: Mnatsakanyan
email: tigran@mna.com
pass: 1234qwer
```

Comment (SADSASD1) [reply Main.java:4]: lav haskaca Tigran jan

**Pull request merged**